core/templates/dev/head/pages/story-viewer-page/chapters-list/story-viewer-chapters-list.directive.html

```diff
@@ -1,7 +1,15 @@
 <div>
+  <background-banner></background-banner>
   <div ng-if="$ctrl.storyPlaythroughObject" class="oppia-story-player-tiles-container">
-    <div class="oppia-story-path-section d-none d-md-block">
-      <svg width="500px" ng-attr-height="<[$ctrl.svgHeight + 'px']>" xmlns="http://www.w3.org/2000/svg">
+    <div class="oppia-page-heading oppia-story-title">
```

Comment:
Why are we moving this stuff into the "story viewer chapters list" if it's not conceptually part of the chapters list?

Reply:
I was following the pattern in the collection player page. Without these here, in the version that was present in develop, the mobile view was behaving weirdly. Hence, I moved these here as well.
Shall I rename the directive then?

Reply:
Yeah, probably a good idea :)

Reply:
Done.